<launch>
	<arg name="rvizconfig" default="$(find urdf_models)/rviz/robot_viewer.rviz"/>

	<param name="robot_description" textfile="$(find urdf_models)/urdf/roboto_gazebo.urdf"/>
	<node pkg="robot_state_publisher" type="state_publisher" name="robot_state_publisher"/>
	
	<param name="use_gui" value="true"/>
	<node pkg="joint_state_publisher" type="joint_state_publisher" name="joint_state_publisher"/>
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(arg rvizconfig)" required="true"/>
</launch>
